crane-cloud/frontend
# Add ability to delete users

# Description
Admin can delete users

## Type of change


## Trello Ticket ID
https://trello.com/c/1JTEd40i

## How Can This Been Tested?
Pull this branch locally and check out out the admin side for the users page select a user and continue with the delete

## Checklist:


# Screenshots
![Screenshot from 2022-05-26 22-10-36](https://user-images.githubusercontent.com/32802973/170563435-832b7cb7-1c3a-4adf-83fd-4cc0a415a02a.png)
![Screenshot from 2022-05-26 22-10-28](https://user-images.githubusercontent.com/32802973/170563449-49ff06f5-0cb2-4874-9bae-85b3311e16d3.png)
